full-stack-intro/client/src/components/Todos.tsx

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+/* eslint-disable @typescript-eslint/no-empty-function -- Remove me */
+import { useEffect, useState } from 'react';
+import { PageTitle } from './PageTitle';
+import { TodoList } from './TodoList';
+import { TodoForm } from './TodoForm';
+
+export type UnsavedTodo = {
+  task: string;
+  isCompleted: boolean;
+};
+export type Todo = UnsavedTodo & {
+  todoId: number;
+};
+
+export function Todos() {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState<unknown>();
+
+  /* Implement useEffect to fetch all todos. Hints are at the bottom of the file. */
+  useEffect(() => {
+    async function getData() {
+      try {
+        const response = await fetch('/api/todos', {
+          method: 'GET',
+          headers: {
```

Comment:
There is no content for a GET request, so there's no need to set the content-type header. Also, GET is the default, so you don't need to set the method, either. For a GET request, you can usually just use `await fetch('/api/todos');`.